src/main/java/com/spotinst/sdkjava/model/api/admin/messageCenter/ApiGetMessage.java

```diff
@@ -0,0 +1,282 @@
+package com.spotinst.sdkjava.model.api.admin.messageCenter;
+
+import com.fasterxml.jackson.annotation.*;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.admin.messageCenter.Metadata;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiGetMessage implements IPartialUpdateEntity {
```

Comment:
Is this response class for the Get message?

Reply:
Yes